README.md

```diff
@@ -12,5 +12,31 @@ An application to execute Flink SQL jobs.
     ```
     minikube image build flink-sql-runner -t flink-sql-runner:latest
     ```
+3. Create a `flink` namespace:
+   ```
+   kubectl create namespace flink
+   ```
+4. Install cert-manager (this creates cert-manager in a namespace called `cert-manager`):
+   ```
+   kubectl create -f https://github.com/jetstack/cert-manager/releases/download/v1.8.2/cert-manager.yaml
+   ```
+5. Deploy Flink Kubernetes Operator:
+   ```
+   helm repo add flink-operator-repo https://downloads.apache.org/flink/flink-kubernetes-operator-<VERSION>/
+   helm install flink-kubernetes-operator flink-operator-repo/flink-kubernetes-operator -n flink
```

Comment:
Wouldn't be better to have there just reference to Flink Operator install guide?